stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/model/dto/DownstreamMessageDto.kt

```diff
@@ -0,0 +1,20 @@
+package io.getstream.chat.android.client.api2.model.dto
+
+import com.squareup.moshi.JsonClass
+import java.util.Date
+
+@JsonClass(generateAdapter = true)
+internal data class DownstreamMessageDto(
+    val id: String,
+    val cid: String,
+    val text: String,
+    val html: String,
+    val parent_id: String?,
+    val command: String?,
+    val user: UserDto,
+    val silent: Boolean,
+    val shadowed: Boolean,
+    val created_at: Date,
+    val updated_at: Date,
+    val deleted_at: Date? = null,
```

Comment:
Just out of curiosity  - why are we adding default value only for `deleted_at` and not for every nullable field?

Reply:
Good catch, an oversight on my part. It makes no sense to have a default `null` value here, anything nullable is treated as optional and will be `null` if not present in the JSON.